tests/mlmodel_openai/test_get_ai_message_ids.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2010 New Relic, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import openai
+from testing_support.fixtures import reset_core_stats_engine
+
+from newrelic.api.background_task import background_task
+from newrelic.api.ml_model import get_ai_message_ids
+from newrelic.api.transaction import add_custom_attribute, current_transaction
+
+_test_openai_chat_completion_messages_1 = (
+    {"role": "system", "content": "You are a scientist."},
+    {"role": "user", "content": "What is 212 degrees Fahrenheit converted to Celsius?"},
+)
+_test_openai_chat_completion_messages_2 = (
+    {"role": "system", "content": "You are a mathematician."},
+    {"role": "user", "content": "What is 1 plus 2?"},
+)
+expected_message_ids_1 = [
```

Comment:
It may be worth testing some cases here where one or some combo of these values are `None`. At a minimum, it's likely conversation ID will be empty in many cases

Reply:
I've added some tests for conversation id being unset. I was originally gonna add more tests into this anticipating bedrock and the response_id being unset but I thought it was better to wait on those until we actually get to bedrock.